app/models/user.rb

```diff
@@ -256,6 +257,11 @@ def default_country_to_usa
       self.country ||= "USA"
     end
 
+    def sanitize_postal_code
+      return unless self.postal_code
```

Comment:
`return if postal_code.blank?`

no need for self
reads better

Reply:
Works for me - changed in  967e880